### ISSUE TITLE
Could Matconvnet train two partially coupled linear ConvNets

### ISSUE THREAD

**Issue by yushuinanrong** [vlfeat/matconvnet#711]
Sorry I open the same issue again because I find the solution I made is not correct.

Basically, I want to train two ConvNets simultaneously. The two networks share a ratio of convolutional filters at each conv layer. So in backpropagate, the graidents for the shared filters would be an average of from two loss functions. 
I don't know if this is possible to do in Matconvnet.
Any idea is greatly appreciated.

Thanks,
Jiawei
